package.json

```diff
@@ -160,8 +160,9 @@
   },
   "files": [
     "build/",
-    "dist/",
-    "flow-typed/",
+    "dist/mapbox-gl*",
```

Comment:
Since there are source maps, are `mapbox-gl-unminified` and `mapbox-gl-dev` required? If not, maybe this can be made more explicit